wurst/systems/commands/Skins.wurst

```diff
@@ -58,12 +58,12 @@ class CustomSkin
 //       See the associated PR for an in-depth explanation.
 //       This is not fixed part of it because skins are currently defunct.
 @compiletime function buildSkins()
-    new CustomSkin(UNIT_BEAST_MASTER,  UNIT_BEAST_MASTER_1,  "skin-bm-1"     )
-    new CustomSkin(UNIT_HUNTER,        UNIT_HUNTER_1,        "skin-hunter-1" )
-    new CustomSkin(UNIT_MAGE,          UNIT_MAGE_1,          "skin-mage-1"   )
-    new CustomSkin(UNIT_MASTER_HEALER, UNIT_MASTER_HEALER_1, "skin-mh-1"     )
-    new CustomSkin(UNIT_BOOSTER,       UNIT_BOOSTER_1,       "skin-booster-1")
-    new CustomSkin(UNIT_ELEMENTALIST,  UNIT_ELEMENTALIST_1,  "skin-ele-1"    )
+    new CustomSkin(UNIT_BEAST_MASTER,      UNIT_BEAST_MASTER_1,  "skin-bm-1"     )
+    new CustomSkin(UNIT_HUNTER,            UNIT_HUNTER_1,        "skin-hunter-1" )
+    new CustomSkin(UNIT_MAGE,              UNIT_MAGE_1,          "skin-mage-1"   )
+    new CustomSkin(UNIT_MASTER_HEALER,     UNIT_MASTER_HEALER_1, "skin-mh-1"     )
+    new CustomSkin(UNIT_BOOSTER,           UNIT_BOOSTER_1,       "skin-booster-1")
+    new CustomSkin(UNIT_ELEMENTALIST_NEW,  UNIT_ELEMENTALIST_1,  "skin-ele-1"    )
```

Comment:
Will this get messed up by your other PR? I thought we were removing this variable.

Reply:
This variable is indeed removed in #585 , I'll check it out once this is merged.